### PR TITLE
add two condition types

### PR DESCRIPTION
**Pull request by lijinhong11**

### CODE REVIEW

File: src/main/java/org/black_ixx/bossshop/core/conditions/BSConditionTypeClientVersion.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        return true;
+    }
+
+    @Override
+    public double getNumber(BSBuy shopitem, BSShopHolder holder, Player p) throws NoSuchFieldException, IllegalAccessException {
+        return getPlayerProtocolVersion(p);
+    }
+
+    private static int getPlayerProtocolVersion(Player player) throws NoSuchFieldException, IllegalAccessException {
+        CraftPlayer craftPlayer = (CraftPlayer) player;
```
Comment (Travja) [inline src/main/java/org/black_ixx/bossshop/core/conditions/BSConditionTypeClientVersion.java:33]: We really can't be using version dependent NMS classes like this. If we want to use NMS, we're going to need to use reflection.

File: src/main/java/org/black_ixx/bossshop/managers/config/ConfigKeyCompleter.java [hunk 2]
```diff
@@ -28,7 +28,7 @@ private static void checkEnUsLanguage(){
         assert is != null;
         File f = new File(ClassManager.manager.getPlugin().getDataFolder(),"lang/en_us.yml");
         File f2;
-        try {f2 = File.createTempFile("en_us-lang-cache","yml");//make cache
+        try {f2 = File.createTempFile("en-US-lang-cache","yml");//make cache
```
Comment (Travja) [inline src/main/java/org/black_ixx/bossshop/managers/config/ConfigKeyCompleter.java:31]: Can we please change this back so it is consistent with en_us.yml a couple lines earlier?

File: src/main/java/org/black_ixx/bossshop/misc/ClassAndVerTools.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    public static boolean classExists(String className) {
+        try {
+            Class.forName(className);
+            return true;
+        } catch (ClassNotFoundException e) {
+            return false;
+        }
+    }
+
+    public static boolean isHighThanOrEquals(int i){
```
Comment (Travja) [inline src/main/java/org/black_ixx/bossshop/misc/ClassAndVerTools.java:15]: A better name could be `isGreaterThan`

**Review by Travja — CHANGES_REQUESTED** [review]

**Comment by lijinhong11** [comment]
Sorry I can use NMS. Can you help me?

**Comment by Travja** [comment]
Is there any reason in particular that we need the player's client version anyway?

**Pull request closed without merging**